### ISSUE TITLE
[FEATURE] NoSlowdown Sneak

### ISSUE THREAD

**Issue by Tcm9** [CCBlueX/LiquidBounce1.8-Issues#3119]
**Describe your feature request.**
You should be able to adjust the speed of sneak.

**Comment by Soulplexis** [comment]
yeah i thought they should make sneak like how ncp noslow is , like how "toggles use" when you hold right click with sword. it should toggle sneak when holding sneak instead.

**Comment by yorik100** [comment]
Good idea, NoSlow sneak with modes like vanilla, NCP, AAC
